app/jobs/dhis2/bangladesh_disaggregated_diabetes_exporter_job.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module Dhis2
+  class BangladeshDisaggregatedDiabetesExporterJob < Dhis2ExporterJob
```

Comment:
This is because the application job inherits from active job, and does not include sidekiq job. 

Active job is compatible with sidekiq but it does not include all the features of sidekiq jobs.